### ISSUE TITLE
[CTRL] + [C] does not work in a Terminal session

### ISSUE THREAD

**Issue by SanPollo** [helloSystem/hello#322]
When using Terminal, and running a command such as ping, it is not possible to stop it by pressing [CTRL] + [C] on the keyboard.

This means that the only way to stop it is by closing the Terminal session.

This is reproducible regardless of how the CMD key is configured in Keyboard Layout preferences.

![CtrlC](https://user-images.githubusercontent.com/6193091/149954267-e964a16e-6904-44f0-85a8-80679489edd1.png)

**Comment by probonopd** [comment]
Need to press Command-SHIFT-C.

**Comment by SanPollo** [comment]
@probonopd surely if Ctrl and Alt are switched, then surely Alt + C should ^C, but if the Ctrl/Alt switching is disabled then Ctrl + C should do ^C ?

I don't have any Mac OS X environments that I can access at the moment (and won't for a couple of weeks), but is the Cmd+Shift+C the normal keypress for ^C in that OS?

**Comment by probonopd** [comment]
Yes. I am leaving this ticket open because the current implementation is probably not ideal but I don't know how to implement it better at the moment without adding undue extra complexity.

**Comment by TriVoxel** [comment]
> @probonopd surely if Ctrl and Alt are switched, then surely Alt + C should ^C, but if the Ctrl/Alt switching is disabled then Ctrl + C should do ^C ?
> 
> I don't have any Mac OS X environments that I can access at the moment (and won't for a couple of weeks), but is the Cmd+Shift+C the normal keypress for ^C in that OS?

On macOS, you can use Command+C to copy selected text in the terminal and Command+V to paste. It is helpful because you don't need to enter a separate command to copy/paste in the terminal. This is how copy/paste works across the whole system, they don't use control. MacOS also has a separate control key and alt key. You can just press Control+C to cancel as Control+C never has a purpose anywhere else in the system. Since standard keyboards don't have a command key, the workaround seems to be to press Command+Shift+C to cancel commands. Perhaps we could get an option in Keyboard settings to use Windows/OS key to copy/paste, etc. and control+c to cancel in terminals and alt could just be alt? Just food for thought @probonopd. Currently, is seems the OS key is used for little more than moving windows about.
